content/docs/command-reference/params/index.md

```diff
@@ -143,6 +143,98 @@ $ dvc run -n train -d logs/ -o users.csv \
           python train.py
 ```
 
+## Examples: Python parameters file
+
+Consider this parameters file in Python format, named `params.py`:
+
+```python
+IS_BOOL: bool = True
+CONST = 5
+
+# All standard variable types are supported
+FLOAT = 0.001
+STR = 'abc'
+DICT = {
+    "a": 1,
+    "b": 2
+}
+LIST = [1, 2, 3]
+SET = {4, 5, 6}
+TUPLE = (10, 100)
+NONE = None
```

Comment:
```suggestion
# All standard types are supported
INT = 5
FLOAT = 0.001
STR = 'abc'
BOOL = True
DICT = {
    "a": 1,
    "b": 2
}
LIST = [1, 2, 3]
SET = {4, 5, 6}
TUPLE = (10, 100)
NONE = None
```
But not sure we want to list them all. People using this feature should know Python 😋 so maybe truncate the list and add a `# etc.`

Reply:
Implies changes in const names throughout the doc